perma_web/perma/utils.py

```diff
@@ -562,19 +562,34 @@ def stream_warc_if_permissible(link, user):
 #
 # Webrecorder Helpers
 #
+DEFAULT_WR_USER = 'perma'
 
-def get_wr_session(request):
-    """
-    Initializes Webrecorder session or retrieves existing session.
-    """
+def get_wr_public_session(request):
```

Comment:
With the new refactoring, it follow same path for public user and private temp user.
I suppose it can be adjusted to not store the WR session cookie in the perma session and log in every time, but don't think it makes much of a difference. For WR to delete the session, you'd also have to add a logout api call. I wanted to minimize the WR api calls made (just two now!). The WR session for perma account is never sent to the user. Even if it somehow were, it just allows access to all the collections that are cached there, which are public anyway.